### DISCUSSION TITLE
use displaced_mesh in the DGKernels

### DISCUSSION THREAD

**Discussion by xiaojbing** [idaholab/moose#20746] [Q&A Meshing] [answered]
I use `displacements = disp_x disp_y` in the `[Mesh] `block, and I also want to use the `use_displaced_mesh = true`  in other block to update the coordinate in the equations. However, in the `[DGKernels]` ,if I don't write `boundary = `, and only write `use_displaced_mesh = true `in the block, it will be crashed; and If I write both the` boundary =` and  `use_displaced_mesh = true` ,it can run. but seems that the results does not make sense. Is there any way to solve this contradictory problem?
![6c372aca812ba2589051fa3eef2e42a](https://user-images.githubusercontent.com/87591264/162610176-e28cb00a-8acc-4998-aedf-f57dce09c02e.png)
![image](https://user-images.githubusercontent.com/87591264/162426851-eccf1b89-7777-475a-a482-e1227dc2ab1c.png)
![image](https://user-images.githubusercontent.com/87591264/162426868-df300512-be25-4a31-ae61-ab15388c438a.png)
![image](https://user-images.githubusercontent.com/87591264/162426896-05f4b451-51fd-40eb-9f05-712fc937328f.png)

**Comment by GiudGiud** [comment, marked as answer]
linked to https://github.com/idaholab/moose/discussions/20583
